<launch>
  
    <node pkg="laser_filters" type="scan_to_scan_filter_chain" name="laser_filter_hokuyo">
      <rosparam command="load" file="$(find otto_sensors)/config/lasershadow.yaml" />
      <remap from="scan" to="/hokuyo/scan" />
      <remap from="scan_filtered" to="hokuyo/scan_filtered" />
    </node>

</launch>
